core/src/test/java/com/klaytn/caver/feature/RpcTest.java

```diff
@@ -511,7 +565,7 @@ public void testGetSha3() throws IOException {
     @Test
     public void testGetId() throws Exception {
         Bytes netVersion = caver.net().getNetworkId().send();
-        assertEquals(netVersion.getResult(), String.valueOf(Scenario.BAOBAB_CHAIN_ID));
+        assertEquals(netVersion.getResult(), String.valueOf(2018));
```

Comment:
What about defining another constant variable for 2018 and using the variable here something like LOCAL_NETWORK_ID?

Reply:
It's a good idea. I'll reflect your opinion!! Thanks